app/controllers/participant_signup_controller.rb

```diff
@@ -0,0 +1,33 @@
+class ParticipantSignupController < ApplicationController
+	before_filter :fetch_participant
```

Comment:
Be sure to clean up indenting all across this file. There's a mix of tabs and spaces and it's kind of hard to read on Github.